### PR TITLE
Fixed crash when running inside container

### LINKED ISSUE #267: Tuned won't start on Centos 7
Installed normal.
`$ systemctl start tuned`
Job for tuned.service failed because the control process exited with error code. See "systemctl status tuned.service" and "journalctl -xe" for details.

`$systemctl status tuned.service`
tuned.service - Dynamic System Tuning Daemon
   Loaded: loaded (/usr/lib/systemd/system/tuned.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Tue 2020-05-19 19:28:37 UTC; 28s ago
     Docs: man:tuned(8)
           man:tuned.conf(5)
           man:tuned-adm(8)
  Process: 10265 ExecStart=/usr/sbin/tuned -l -P (code=exited, status=1/FAILURE)
 Main PID: 10265 (code=exited, status=1/FAILURE)

May 19 19:28:37 any2.domain.com tuned[10265]: reraise(exc_value, traceback)
May 19 19:28:37 any2.domain.com tuned[10265]: File "/usr/lib/python2.7/site-packages/pyudev/monitor.py", line 275, in set_receive_buffer_size
May 19 19:28:37 any2.domain.com tuned[10265]: self._libudev.udev_monitor_set_receive_buffer_size(self, size)
May 19 19:28:37 any2.domain.com tuned[10265]: File "/usr/lib/python2.7/site-packages/pyudev/_libudev.py", line 219, in check_errno
May 19 19:28:37 any2.domain.com tuned[10265]: raise exception_from_errno(errno)
May 19 19:28:37 any2.domain.com tuned[10265]: EnvironmentError: [Errno 1] Operation not permitted: None
May 19 19:28:37 any2.domain.com systemd[1]: tuned.service: main process exited, code=exited, status=1/FAILURE
May 19 19:28:37 any2.domain.com systemd[1]: Failed to start Dynamic System Tuning Daemon.
May 19 19:28:37 any2.domain.com systemd[1]: Unit tuned.service entered failed state.
May 19 19:28:37 any2.domain.com systemd[1]: tuned.service failed.

Please help, thank.

### PR DESCRIPTION
**Pull request by yarda**
Fixes #267

Signed-off-by: Jaroslav Škarvada <jskarvad@redhat.com>

### CODE REVIEW

File: tuned/hardware/inventory.py [hunk 1]
```diff
@@ -25,7 +25,11 @@ def __init__(self, udev_context=None, udev_monitor_cls=None, monitor_observer_fa
 			buffer_size = consts.CFG_DEF_UDEV_BUFFER_SIZE
 
 		if (set_receive_buffer_size):
-			self._udev_monitor.set_receive_buffer_size(buffer_size)
+			try:
+				self._udev_monitor.set_receive_buffer_size(buffer_size)
+			except EnvironmentError:
+				log.warn("cannot set udev monitor receive buffer size, we are probably running inside " + \
```
Comment (olysonek) [inline tuned/hardware/inventory.py:31]: The backslash at the end of the line is unnecessary because it's between parentheses.

Comment (olysonek) [reply tuned/hardware/inventory.py]: Sorry, the plus is unnecessary too. But don't worry about it :D.

Comment (yarda) [reply tuned/hardware/inventory.py]: Do you want me to remove it? Personally, I don't care, but maybe with the "+" it seems more readable to me. IMHO similar constructs are already in the code.

Comment (olysonek) [reply tuned/hardware/inventory.py]: OK, we can leave it.

**Comment by yarda** [comment]
Thanks, merging.

**Pull request merged**